qa/sc_closed_forger.py

```diff
@@ -194,6 +195,7 @@ def run_test(self):
         except SCAPIException as e:
             logging.info("Expected SCAPIException: " + e.error)
             error_occur = True
+
         assert_true(error_occur, "Try to send an openStake transaction with empty output proposition")
```

Comment:
Consider to remove `assert_true(error_occur...` and to put a `fail` after `createOpenStakeTransaction`. This way, calling assert_true can be avoided when SCAPIException is actually thrown.
The same comment can be applied in other files when the `except` clause doesn't catch a generic Exception, as in this case.